README.md

```diff
@@ -18,7 +18,11 @@ The Node.js version to install. "0.10" is the default and works on all supported
 
     nodejs_install_npm_user: "{{ ansible_ssh_user }}"
 
-The user for whom the npm packages will be installed can be set here, this defaults to ansible_ssh_user
+The user for whom the npm packages will be installed can be set here, this defaults to ansible_user
```

Comment:
Ansible 2.0 has deprecated the “ssh” from ansible_ssh_user, ansible_ssh_host, and ansible_ssh_port to become ansible_user, ansible_host -- http://docs.ansible.com/ansible/faq.html